### ISSUE TITLE
On .csv import, weights are not being imported (all set to 1.0)

### ISSUE THREAD

**Issue by richgor** [gephi/gephi#1254]
I am trying to create a new Gephi project by importing a nodes CSV and an edges CSV.

My edges CSV has weights ranging from 3 to 1,267.  But when I import the CSV, all the weights are converted to 1.0.

![screen shot 2015-12-28 at 3 02 00 pm](https://cloud.githubusercontent.com/assets/1087501/12025378/f8b10a0c-ad73-11e5-9d31-5613b8069b24.png)

**Comment by eduramiba** [comment]
Hi,
Did you install updates in the update center?

In case you did, is this the same as issue #1251 (repeated edge's weights not summed)?

**Comment by richgor** [comment]
I did not install updates, as per our parallel conversation at https://github.com/gephi/gephi/issues/1253 ... so that's a good argument for installing the updates. I guess you can mark this one as Closed. But as noted in that thread, if I install the updates, Gephi hangs when I switch from Data Laboratory to Overview.  

Good to know that you've fixed the import problem with weights.  But for now my bigger problem is that I can't do *anything* with Gephi if I have the updates installed.  So for now I guess I will live with the problem with weights.

**Comment by eduramiba** [comment]
Right, sorry for the inconvenience and I hope we can get the JOGL this fixed soon.

PD: You could try to only mark for update data laboratory modules, and keep other modules unchecked. That will fix the weight problem. Then see if you still get the hang problem or it's related to the other not updated modules.

**Comment by richgor** [comment]
Good suggestion.  Will try that if I have time - right now I'm scrambling to get a course ready to launch next week.

**Comment by richgor** [comment]
@eduramiba ... just to report: I was able to update the Data Laboratory module (*without*) the Appearance API, the Visualization Module and the Data Laboratory Plugin.  With these updates, the import problem (weights not being imported) was solved -- AND I do not have the "hang" problem when switching from Data Laboratory to Overview.  

This suggests that whatever is causing the "hang" problem is somehow related to either the Appearance API or the Visualization Module.
